postgres/server/image.sls

```diff
@@ -5,20 +5,22 @@
 
 {%- if postgres.bake_image %}
 
-include:
```

Comment:
If you have removed this here, probably you wanted to include the state in parent `postgres.server` SLS?
Of course need to deal with conflicting states.

Reply:
@vutny it IS included in postgres/server/init.sls (line 5).